### ISSUE TITLE
2019-11-20 tc012 M1SS comp engine error when parameterset set to "PARAMETERDISPLAYLIST"

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#42]
### Issue
When using tc012 and using the value of `"PARAMETERDISPLAYLIST"` for `parameterset` the computation engine crashes. This does not cause an error for tc010, tc011, tc013 or tc014. Errors for tc015 and tc016 are documented separately due to their unique error messages.

### Error

```
parameterset <- "PARAMETERDISPLAYLIST"
```
```
Error in `[.data.frame`(computation_df, , flag_df$VAR[f]) : 
  undefined columns selected
```

No option for traceback is provided for this error.

**Comment by jhhughes256** [comment]
### Update

This issue has been replicated with tc071, which provides a traceback included below:

```
Error in `[.data.frame`(computation_df, , flag_df$VAR[f]) : 
  undefined columns selected 
5. stop("undefined columns selected") 
4. `[.data.frame`(computation_df, , flag_df$VAR[f]) at run_M1_SS_computation.R#2669
3. computation_df[, flag_df$VAR[f]] at run_M1_SS_computation.R#2669
2. run_M1_SS_computation(data = merged_data, map = map_data, method = method, 
    model_regex = model_regex, parameter_list = parameter_list, 
    return_list = return_list) at run_computation.R#2223
1. run_computation(data = d, map = mct, flag = flags, parameterset = parameterset) 
```

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This issue has been fixed as of 02b2470.
